<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QStackedWidget" name="main_window_stack_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QFrame" name="frame_15">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Select data / sample</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sample_data_combo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_7">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="6">
            <widget class="Line" name="line_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QCheckBox" name="sample_agents_check">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>1677215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Agent(s)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="sample_step_check">
             <property name="text">
              <string>Step range</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QWidget" name="agent_sample_combo_widget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_29">
                <property name="text">
                 <string>start:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sample_step_start_spinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_30">
                <property name="text">
                 <string> end:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sample_step_end_spinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QPushButton" name="save_sample_button">
             <property name="text">
              <string>SAVE</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="sample_output"/>
        </item>
        <item>
         <widget class="QFrame" name="frame_20">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QPushButton" name="sample_help_button">
             <property name="text">
              <string>Sample Help</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>641</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>simulation</string>
    </property>
    <addaction name="actionrun_sim"/>
   </widget>
   <widget class="QMenu" name="menudata">
    <property name="title">
     <string>data</string>
    </property>
    <addaction name="actionview_data"/>
    <addaction name="actionquery"/>
   </widget>
   <widget class="QMenu" name="menuwindow">
    <property name="title">
     <string>plot</string>
    </property>
    <addaction name="actionplot"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionmanual"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menudata"/>
   <addaction name="menuwindow"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelect_load_config">
   <property name="text">
    <string>load config</string>
   </property>
  </action>
  <action name="actionview_data">
   <property name="text">
    <string>view table of stored data</string>
   </property>
  </action>
  <action name="actionquery_data">
   <property name="text">
    <string>query data</string>
   </property>
  </action>
  <action name="actioncreate_samples">
   <property name="text">
    <string>create samples</string>
   </property>
  </action>
  <action name="actioncreate_plot">
   <property name="text">
    <string>create plot</string>
   </property>
  </action>
  <action name="actionview_samples">
   <property name="text">
    <string>view samples</string>
   </property>
  </action>
  <action name="actionload_data_from_file">
   <property name="text">
    <string>load from file</string>
   </property>
  </action>
  <action name="actionquery_data_page">
   <property name="text">
    <string>query data</string>
   </property>
  </action>
  <action name="actionmain">
   <property name="text">
    <string>main </string>
   </property>
  </action>
  <action name="actionplot">
   <property name="text">
    <string>plot query results</string>
   </property>
  </action>
  <action name="actionsample">
   <property name="text">
    <string>sample</string>
   </property>
  </action>
  <action name="actionrun_sim">
   <property name="text">
    <string>view simulation info</string>
   </property>
  </action>
  <action name="actionquery">
   <property name="text">
    <string>create query from data/sample</string>
   </property>
  </action>
  <action name="actionsample_data">
   <property name="text">
    <string>create sample from data</string>
   </property>
  </action>
  <action name="actionmanual">
   <property name="text">
    <string>manual</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
